cmd/cluster-network-operator/leader.go

```diff
@@ -29,26 +27,26 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
-var log = logf.Log.WithName("leader")
+var logger = logf.Log.WithName("leader")
```

Comment:
rather than using the controller-runtime logging, it would be good to just make this files use `"log"` like the rest of CNO. (I mean, we should make the rest of CNO use klog really, but, this would at least get rid of one logger.) Note that if you do this then `cmd/cluster-network-operator/log.go` can go away

Reply:
ack